src/routes/safe/components/Settings/SpendingLimit/index.tsx

```diff
@@ -0,0 +1,72 @@
+import { Button, Text, Title } from '@gnosis.pm/safe-react-components'
+import React from 'react'
+import { useSelector } from 'react-redux'
+import styled from 'styled-components'
+
+import Block from 'src/components/layout/Block'
+import Col from 'src/components/layout/Col'
+import Row from 'src/components/layout/Row'
+import { safeSpendingLimitsSelector } from 'src/logic/safe/store/selectors'
+import { grantedSelector } from 'src/routes/safe/container/selector'
+
+import LimitsTable from './LimitsTable'
+import { getSpendingLimitData } from './LimitsTable/dataFetcher'
+import NewLimitModal from './NewLimitModal'
+import NewLimitSteps from './NewLimitSteps'
+import { useStyles } from './style'
+
+const InfoText = styled(Text)`
```

Comment:
Same as before, we are mixing `useStyles` with `styledComponents` on the same file